playground/mocks/data/idp/idx/identify.json

```diff
@@ -71,5 +71,13 @@
            "mutable":false
         }
      ]
-  }
+  },
+  "app": {
+      "type": "object",
```

Comment:
is this value always returned in the response? If not, would be great to also test case where there is no appId

Reply:
Yes most idx responses should include "app" objects, despite many of our mocks missing them (https://okta.slack.com/archives/CCA77QVL2/p1623284224193900), except for failure responses where app context cannot be established.